doc/source/getting_started/notebooks/notebook2python.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Well then we cant generate them locally on windows or mac. I felt like the whole idea with python etc is that we stay platform independent, even for developers.